src/js/jsx/sections/libraries/LibraryList.jsx

```diff
@@ -83,9 +83,11 @@ define(function (require, exports, module) {
                 libraryName = this.props.selected.name;
             }
 
+            var listID = "libraries-" + this.props.document.id;
+
             return (
                 <Datalist
-                    list={"libraries"}
+                    list={listID}
```

Comment:
Pinging @shaoshing to make sure he's aware of these changes and he rebases when these are merged.